src/wallet/wallet.ts

```diff
@@ -25,6 +25,12 @@ import {
 } from './types';
 import { sortedJsonStringify } from '@cosmjs/amino/build/signdoc';
 
+export interface SignTransactionOptions {
+  chainId?: string;
```

Comment:
I think we should not have the chain ID configurable as a sign option, because the provider itself is "tied' to a network in a way (it does a status call via its own configured RPC)

Reply:
What do you think?